scripts/biomedical/diseaseOntology/disease_ontology.tmcf

```diff
@@ -0,0 +1,24 @@
+Node: E:DiseaseOntology->E1
+typeOf: dcs:Disease
+dcid: C:DiseaseOntology->dcid
+parent: C:DiseaseOntology->subClassOf
+diseaseDescription: C:DiseaseOntology->description 
+alternativeDOIDs : C:DiseaseOntology->hasAlternativeId 
+diseaseSynonym: C:DiseaseOntology->hasExactSynonym
+commonName: C:DiseaseOntology->label
+icdoID: C:DiseaseOntology->ICDO
+meshID: C:DiseaseOntology->MESH
+nciID: C:DiseaseOntology->NCI
+snowmedctusID: C:DiseaseOntology->SNOMEDCTUS20200901
```

Comment:
are these SNOMEDCTUS20200901 different columns in the file? Why are they stored under such odd column names?

Reply:
the columns represent different version of the snow med, so SNOMEDCTUS20200901 refers to the 09/01/2020 version of the data